dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/Java8IncompatibleReferenceCheck.java

```diff
@@ -109,7 +109,7 @@ public static void main(String[] arguments)
     StringBuilder message = new StringBuilder();
     message.append(
         "The following artifacts contain bad references to classes in 'java' package,"
```

Comment:
delete "bad"
perhaps "classes in 'java' package" --> classes in the core library
which do not work for --> which are not present in

Reply:
Updated.